backend/core/src/listings/dto/listing.dto.ts

```diff
@@ -812,6 +812,15 @@ export class ListingFilterParams extends BaseFilter {
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   [ListingFilterKeys.seniorHousing]?: boolean;
 
+  @Expose()
+  @ApiProperty({
+    type: Boolean,
+    example: "true",
+    required: false,
+  })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
```

Comment:
Can you add `@IsBooleanString({ groups: [ValidationsGroupsEnum.default] })` here, for validation?